Extension/src/LanguageServer/settings.ts

```diff
@@ -712,29 +712,21 @@ export class CppSettings extends Settings {
                         return true;
                     }
                 }
-                if (editorConfigSettings.root !== undefined) {
-                    if (typeof editorConfigSettings.root === "boolean") {
-                        if (editorConfigSettings.root) {
-                            return true;
-                        }
-                    } else if (typeof editorConfigSettings.root === "string") {
-                        if (editorConfigSettings.root.toLowerCase() === "true") {
-                            return true;
-                        }
-                    }
-                }
-            } else {
-                const clangFormatPath1: string = path.join(parentPath, ".clang-format");
-                if (fs.existsSync(clangFormatPath1)) {
-                    return true;
-                } else {
-                    const clangFormatPath2: string = path.join(parentPath, "_clang-format");
-                    if (fs.existsSync(clangFormatPath2)) {
-                        return true;
-                    }
+                switch (typeof editorConfigSettings.root) {
```

Comment:
I don't think this should be backed out.  This wasn't part of @elahehrashedi's change.

Reply:
Just `configurationProvider.ts`

Reply:
Nevermind. it looks like there was some other commit that changed this.